hazelcast/src/main/java/com/hazelcast/config/ManagementCenterConfig.java

```diff
@@ -84,6 +86,28 @@ public boolean isConsoleEnabled() {
         return consoleEnabled;
     }
 
+    /**
+     * Enables/disables data access for Management Center.
+     * Management Center can't access the data if at least one member has the data access disabled.
+     * <p>
+     * Default value for this config element is {@code true}.
+     *
+     * @param dataAccessEnabled {@code true} to allow data access for Management Center, {@code false} to disallow
+     * @return this management center config instance
+     * @since 5.1
+     */
+    public ManagementCenterConfig setDataAccessEnabled(boolean dataAccessEnabled) {
+        this.dataAccessEnabled = dataAccessEnabled;
+        return this;
+    }
+
+    /**
+     * Returns if data access for Management Center is allowed ({@code true}) or disallowed ({@code false}).
```

Comment:
nit: let's elaborate on that "data access" is SQL browser or Map Browser (or just delete this javadoc, since it doesn't convey more information than the method signature).

Reply:
agree, updated